core/common/src/DateTimeUnit.kt

```diff
@@ -91,10 +93,11 @@ public sealed class DateTimeUnit {
     }
 
     /**
-     * The date-time unit equal to some number of days or months.
+     * A date-time unit equal to some number of days or months.
      *
-     * Unless used for operations on dates, measuring with such units requires knowledge of the time zone, as it is
-     * otherwise unknown when one date ends and another one begins.
+     * Operations involving `DateBased` units are performed with dates. The same operations on [Instants][Instant]
+     * require a [TimeZone] to find the corresponding [LocalDateTimes][LocalDateTime] first and then to perform
```

Comment:
> `require a [TimeZone] to find the corresponding [LocalDateTimes][LocalDateTime] first and then to perform`
`require a [TimeZone] to find the corresponding [LocalDateTimes][LocalDateTime] first to perform`

Otherwise, it looks like the time zone is also needed for the latter.

Reply:
Applied